product_customer_code/models/product_customer_code.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+
+from openerp import api, fields, models
+
+
+class ProductCustomerCode(models.Model):
+
+    _name = "product.customer.code"
+    _description = "Add manies Code of Customer's"
+
+    _rec_name = 'product_code'
+
+    product_code = fields.Char('Customer Product Code', required=True,
+                               index=True, help="""This customer's product
+                               code will be used when searching into a request
+                               for quotation.""")
+    product_name = fields.Char('Customer Product Name', help="""This customer's
```

Comment:
Use just one double-quote to help because the newline could be displayed from gui